systemtest/driver/flow.py

```diff
@@ -80,27 +82,60 @@ def check_max_agents(flow_dict, max_agents_dict):
     return success, agents
 
 
-def check_door_usage(flow_dict, num_agents):
-    success = True
-
-    # All pedestrian should leave through the main exit
-    exit_usage = max(flow_dict.values())  # maximum value
-    if exit_usage != num_agents:
-        success = False
-        logging.error(
-            "Number of agents exiting the simulation ({}) does not match number of agents in the simulation ({})".format(
-                exit_usage, num_agents
-            )
-        )
-
-    # All pedestirans should pass one of the other doors
-    other_door_usage = sum(flow_dict.values()) - exit_usage
-    if other_door_usage != num_agents:
-        success = False
-        logging.error(
-            "Number of agents using other doors ({}) does not match number of agents in the simulation ({})".format(
-                other_door_usage, num_agents
+def read_starting_times(events_file: Path):
+    tree = ET.parse(events_file)
+    root = tree.getroot()
+    starting_times = {}
+    for event in root.iter("event"):
+        if event.attrib["state"] == "open" or event.attrib["state"] == "reset":
+            starting_times.setdefault(int(event.attrib["id"]), []).append(
+                float(event.attrib["time"])
```

Comment:
The code in `driver` tries to offer functionality for each input file with a `.py` file of the same name, e.g. `ini file.py` `geometry.py` etc... Since this is extracted from `events.xml` I would like to see this code moved to `events.py`

Reply:
done